src/fastfield/mod.rs

```diff
@@ -244,6 +72,15 @@ fn value_to_u64(value: &Value) -> u64 {
     }
 }
 
+/// Trait for `BytesFastFieldReader` and `MultiValuedFastFieldReader` to return the length of data
+/// for a doc_id
+pub trait MultiValueLength {
```

Comment:
what is the point of this trait?

Reply:
To return the number of values for a DocId. It's e.g. used in the merge
`fn write_1_n_fast_field_idx_generic<T: MultiValueLength>(`